### PR TITLE
[DC] Dedup- compatibility with outlet change, url check fix/addition, article scrapper

### PR DESCRIPTION
**Pull request by zackyl**
## Summary
* what changes have you made?
Updated dedup to be compatible with the new outlet that's needed by the MetadataExtractor
changed articleid to be at the bottom of ArticleData shared object to avoid errors instantiating the object with params in wrong order
Added testing dedup script (Jarrod)
manually scrapped some articles
added a batch url dedup to run.py
changed unshortern URL to send url request with browser agent to avoid rejection and wrapped with try catch
added new script based off run.py that scraps new articles into json format in new_test

changed files of interest in commit d4957d6cb5e3870c8db98b8c6ee5e2f30d3cecf8:
DataCollection/deduping/deduping.py
DataCollection/deduping/deduping_test.py
DataCollection/deduping/mock_db.py
DataCollection/deduping/run_pull.py
DataCollection/run.py

everything else should be json files 

## Testing Plan
* what have you done to test these changes?
run.py runs and it appears to work and also get deduped (number of articles goes down after url dedup batch)
new scrapper works and added a new json file for each scrapped article

## Affected Services
* which services are your changes affecting (ex: data collection, etc.)
run.py is changed and now uses batch url dedup before anything else
first parameter of ArticleData is now url again
unshortern url from the url checker should be more reliable 
a lot more test data is available at deduping/test_data/new_test

## Reviewers
@jarrodddunne

### CODE REVIEW

**Comment by zackyl** [comment]
[url check integration](https://trello.com/c/iUhdQnex/11-url-check-integration)

File: DataCollection/deduping/run_pull.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
+import json
```
Comment (caiismyname) [inline DataCollection/deduping/run_pull.py]: I'm not sure I understand the purpose of this. Is this a script for you guys to create training data?

File: DataCollection/deduping/test_data/new/@RiceUNews @rimaanabtawi @Dr_U.033453.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (caiismyname) [inline DataCollection/deduping/test_data/new/@RiceUNews @rimaanabtawi @Dr_U.033453.json]: This is an unfair advantage in the "lines of code contributed race" 😆

**Comment by zackyl** [comment]
outdated

**Pull request closed without merging**